pkg/apis/projectcalico/v3/calico/policy.go

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) 2017-2018,2020-2021 Tigera, Inc. All rights reserved.
```

Comment:
I might name this file `policy_common.go` or something like that - we should have `networkpolicy.go` and `globalnetworkpolicy.go` for the specific kinds.